### DISCUSSION TITLE
SQL Agent job list - preserve list filter / sort selections

### DISCUSSION THREAD

**Discussion by Aardvark34** [microsoft/azuredatastudio#22401] [Ideas]
The SQL Agent extension job list is quite helpful. I particularly like the display of the previous run duration bar graph. I would like to suggest that any filters and/or sorts that are applied to the list remain in effect when returning to the job list after viewing an individual job detail. Also, it would be very helpful if the job just viewed would retain the focus or highlight, instead of reverting to the top of the job list.

**Comment by erinstellato-ms** [comment]
Hi @Aardvark34 thanks for your feedback.  It would be great if you could log this as a feature request in the repo (https://github.com/microsoft/azuredatastudio/issues) so that it can be reviewed and added to our backlog as an improvement.  Thanks!
